### PR TITLE
Next step in turning down the Helm Hub

### PR DESCRIPTION
**Pull request by mattfarina**
- The CircleCI config used to deploy the Helm Hub has been removed.
  If we need to undo this and restore the functionlaity, this file
  can be pulled from the git history.
- The readme has been updated with the details of the move.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,65 +1,50 @@
 # Helm Hub
 
-The Helm Hub repository contains the documentation and some configuration for the
-distributed public repository search hosted by Helm that can be found at https://hub.helm.sh
+The Helm Hub repository contained the documentation and some configuration for the
+distributed public repository search hosted by Helm that used to be found at https://hub.helm.sh
 
-## Distributed Search
+## NOTICE: Redirect To Artifact Hub
```
Comment (scottrigby) [inline README.md:6]: I've put these in various places for deprecation notices, and it seems to attract attention. WDYT?

```suggestion
## ⚠️ NOTICE: Redirect To Artifact Hub
```

File: README.md [hunk 2]
```diff
@@ -1,65 +1,50 @@
...
+provides a better experience to that of the Helm Hub.
 
-Discovering the charts hosted by the Helm project via the [stable and incubator](https://github.com/helm/charts)
-repositories has been relatively easy. We want to go further and make it possible
-to discover the charts and repositories being hosted by others.
+## Claiming Your Repo In The Artifact Hub
 
-The Hub provides a means to more easily find charts that are hosted outside the Helm
-project. This has implications for those who build and distribute charts including:
+If you had a repository in the Helm Hub but had not listed it in the Artifact
```
Comment (scottrigby) [inline README.md:15]: Minor change. Closer to how the blog post noted this

```suggestion
If you had a repository in the Helm Hub but had not already listed it in the Artifact
```

File: README.md [hunk 3]
```diff
@@ -1,65 +1,50 @@
...
+Hub, it was automatically listed. Everything that as in the Helm Hub should be
+discoverable in the Artifact Hub.
 
-- They can manage the workflow around releases themselves
-- They can track metrics around the use of their charts
+There are two ways to claim your listing so that you can manage it on the Artifact
+Hub:
 
-## Artifact Hub Migration
+1. If you create an account on the Artifact Hub there is a method to claim your
```
Comment (scottrigby) [inline README.md:22]: I seem to remember a link to artifacthub/hub documentation on this. If I'm remembering that right, maybe add a link to that here?

**Review by scottrigby — COMMENTED** [review]
All looks good! some questions/suggestions

File: README.md (hunk 3 above)
Comment (mattfarina) [reply README.md]: Done

**Review by scottrigby — APPROVED** [review]
🙌

**Comment by scottrigby** [comment]
Cross-linking https://github.com/helm/hub/issues/439

**Pull request merged**